layers/+window-management/exwm/funcs.el

```diff
@@ -78,6 +78,68 @@ Can show completions at point for COMMAND using helm"
   (interactive)
   (start-process "" nil exwm-locking-command))
 
+
+(defvar exwm//autostart-process-list nil
+  "List of processes run during autostart.")
```

Comment:
- Variable should be defined in config.el.
- Documentation string is not a complete sentence. Grammar error.
- It seems that this is a list of strings used as shell commands. This needs to be documented as well.